app/prj.conf

```diff
@@ -9,14 +9,6 @@ CONFIG_BUILD_OUTPUT_BIN=n
 # and usage in xtensa-build-zephyr.py
 CONFIG_BUILD_OUTPUT_STRIPPED=y
 
-# Zephyr has hardcoded this to 'y' because it saves CI some disk space
-# when compiling hundreds of different zephyr/tests/, each in a separate
-# build directory. But when compiling just one "sof/app" it saves
-# practically no space, breaks the incremental build and can make it
-# more difficult to troubleshoot some build issues. So, reset it to the
-# default value.
-CONFIG_CLEANUP_INTERMEDIATE_FILES=n
```

Comment:
config CLEANUP_INTERMEDIATE_FILES
        bool "Remove all intermediate files"
        help
          Delete intermediate files to save space and cleanup clutter resulting
          from the build process. Note this breaks incremental builds, west spdx
          (Software Bill of Material generation), and maybe others.

wonder why there is no default n in the definition? @kv2019i @lyakh

Reply:
@btian1 I think "n" is the default

Reply:
Yes "n" is the default:
https://www.kernel.org/doc/html/latest/kbuild/kconfig-language.html#menu-attributes

But it used to be overriden in Zephyr. And then overriden again in SOF. Now that's all going away.

Click on the referenced commits.